web/assets/js/main_out.js

```diff
@@ -532,55 +555,23 @@
     const eatSound = new Sound("./assets/sound/eat.mp3", 0.5, 10);
     const pelletSound = new Sound("./assets/sound/pellet.mp3", 0.5, 10);
 
-    if (null !== wHandle.localStorage) {
-        wjQuery(window).load(function() {
-            wjQuery(".save").each(function() {
-                var id = wjQuery(this).data("box-id");
-                var value = wHandle.localStorage.getItem("checkbox-" + id);
-                if (value && value == "true" && 0 != id) {
-                    wjQuery(this).prop("checked", "true");
-                    wjQuery(this).trigger("change");
-                } else if ((id == 0 || id == 11) && value != null)
-                    wjQuery(this).val(value);
-            });
-            wjQuery(".save").change(function() {
-                var id = wjQuery(this).data("box-id");
-                var value = (id == 0 || id == 11) ? wjQuery(this).val() : wjQuery(this).prop("checked");
-                wHandle.localStorage.setItem("checkbox-" + id, value);
-            });
-        });
-    }
-    if(typeof allSkins == 'undefined') {
-      wjQuery.ajax({
-          type: "POST",
-          dataType: "json",
-          url: "checkdir.php",
-          data: { "action": "getSkins" },
-          success: function(data) {
-              var stamp = Date.now();
-              var response = JSON.parse(data.names);
-              for (var i = 0; i < response.length; i++)
-                  knownSkins[response[i]] = stamp;
-              for (var i in knownSkins)
-                  if (knownSkins[i] !== stamp) delete knownSkins[i];
-          }
-      });
-    } else {
-      var stamp = Date.now();
-      for (var i = 0; i < allSkins.length; i++)
-          knownSkins[allSkins[i]] = stamp;
-      for (var i in knownSkins)
-          if (knownSkins[i] !== stamp) delete knownSkins[i];
-    }
+    request("skinList.txt", function(data) {
+        var skins = data.split(",");
+        var stamp = Date.now();
+        for (var i = 0; i < skins.length; i++)
+            knownSkins[skins[i]] = stamp;
+        for (var i in knownSkins)
+            if (knownSkins[i] !== stamp) delete knownSkins[i];
+    });
 
     function hideESCOverlay() {
         escOverlayShown = false;
-        byId('overlays').style.display = 'none';
+        byId("overlays").hide();
     }
     function showESCOverlay() {
         escOverlayShown = true;
-        wjQuery("#overlays").fadeIn(300);
-    }
+        byId("overlays").show();
```

Comment:
it'd be nice to keep fade in some way.. css animations?

Reply:
The fade is implemented with the CSS `transition` property in `index.css`, and that's why in the `Element.prototype.show` function it sets the opacity to 1.